Tests/Packet++Test/Tests/TcpTests.cpp

```diff
@@ -225,23 +225,23 @@ PTF_TEST_CASE(TcpPacketCreation2)
 	tcpLayer.getTcpHeader()->synFlag = 1;
 	tcpLayer.getTcpHeader()->windowSize = htobe16(14600);
 
-	PTF_ASSERT_TRUE(tcpLayer.addTcpOption(pcpp::TcpOptionBuilder(pcpp::TcpOptionBuilder::NOP)).isNotNull());
+	PTF_ASSERT_TRUE(tcpLayer.addTcpOption(pcpp::TcpOptionBuilder(pcpp::TcpOptionBuilder::NopEolOptionTypes::NOP)).isNotNull());
```

Comment:
ditto: maybe we can keep the previous version also to make sure we preserve backward compatilitiy